www/src/components/Preview/styles.module.css

```
@@ -0,0 +1,54 @@
+}
+
+.toggle:hover {
+  color: var(--ifm-color-emphasis-900);
+}
+
+.code {
+  border-top: 1px solid var(--ifm-color-emphasis-200);
+}
+
+/* The code block already carries the page's outer spacing; strip its stray bottom margin. */
+.code :global(.theme-code-block) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)